base/hosts.h

```diff
@@ -28,6 +28,8 @@
 #ifndef _GVM_HOSTS_H
 #define _GVM_HOSTS_H
 
+#define FEATURE_HOSTS_ALLOWED_ONLY 1
```

Comment:
Briefly document why this is needed and what it does in code.

Reply:
I added above this line. Therefore, it is not shown as `outdated`

Reply:
```suggestion
#define FEATURE_HOSTS_ALLOWED_ONLY
```
You also don't need to set a value

Reply:
Although it is true that it isn't required when using `#ifdef` I think it is more clear to set it to 1 since that also allows runtime if like:
```
if (#FEATURE_HOSTS_ALLOWED)
   ...
```